### PR TITLE
Docs/academy link

### PR DESCRIPTION
**Pull request by Mcastres**
#### Description of what you did:

I added a link to the academy in the navbar of the documentation

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@Mcastres tons of other commits here, I think you started this off the master branch which hasn't been merged into the documentation branch. Can you please rebase this?

File: examples/getstarted/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "getstarted",
   "private": true,
-  "version": "3.2.2",
+  "version": "3.2.3",
```
Comment (derrickmehaffy) [inline examples/getstarted/package.json:4]: Other commits

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

File: examples/getstarted/package.json (hunk 1 above)
Comment (Mcastres) [reply examples/getstarted/package.json:4]: Hey Derrick, sorry I lost the thing here. What should I do exactly to merge this?

**Comment by derrickmehaffy** [comment]
Closing per discussion in slack, adding it to the README.md instead.

**Pull request closed without merging**